docs/intro.md

```diff
@@ -0,0 +1,174 @@
+###Intro to Kubernetes on Mesos
```

Comment:
It goes easier on the eye if some spacing is added before the text (IMO). `### Intro (...)`